ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -103,100 +178,111 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
-                props.cgnDetails?.status === StatusEnum.ACTIVATED && (
-                  <View>
-                    <FooterWithButtons
-                      type="SingleButton"
-                      primary={{
-                        type: "Solid",
-                        buttonProps: {
-                          label: I18n.t("bonus.cgn.detail.cta.buyers"),
-                          accessibilityLabel: I18n.t(
-                            "bonus.cgn.detail.cta.buyers"
-                          ),
-                          onPress: onPressShowCgnDiscounts
-                        }
-                      }}
-                    />
-                  </View>
-                )}
-            </>
-          )}
-        </SafeAreaView>
-      </BaseScreenComponent>
+            {props.cgnDetails && (
+              <>
+                <VSpacer size={48} />
+                <VSpacer size={40} />
+                <CgnCardComponent
+                  cgnDetails={props.cgnDetails}
+                  onCardLoadEnd={onCardLoadEnd}
+                />
+              </>
+            )}
+            <View style={[IOStyles.flex, IOStyles.horizontalContentPadding]}>
+              {CardRevoked.is(props.cgnDetails) && (
+                <Alert
+                  variant="error"
+                  content={I18n.t("bonus.cgn.detail.information.revoked", {
+                    reason: props.cgnDetails.revocation_reason
+                  })}
+                />
+              )}
+              {CardExpired.is(props.cgnDetails) && (
+                <Alert
+                  variant="error"
+                  content={I18n.t("bonus.cgn.detail.information.expired", {
+                    date: formatDateAsShortFormat(
+                      props.cgnDetails.expiration_date
+                    )
+                  })}
+                />
+              )}
+              {/* Ownership block rendering owner's fiscal code */}
+              <CgnOwnershipInformation />
+              <VSpacer size={16} />
+              {props.cgnDetails && (
+                // Renders status information including activation and expiring date and a badge that represents the CGN status
+                // ACTIVATED - EXPIRED - REVOKED
+                <CgnStatusDetail cgnDetail={props.cgnDetails} />
+              )}
+              {canDisplayEycaDetails && <EycaDetailComponent />}
+              <VSpacer size={24} />
+              {CardActivated.is(props.cgnDetails) && <CgnUnsubscribe />}
+              <VSpacer size={40} />
+            </View>
+          </Animated.ScrollView>
+          <SectionStatusComponent sectionKey={"cgn"} />
+          {props.isCgnEnabled &&
+            props.cgnDetails?.status === StatusEnum.ACTIVATED && (
+              <GradientBottomActions
+                primaryActionProps={{
+                  label: I18n.t("bonus.cgn.detail.cta.buyers"),
+                  accessibilityLabel: I18n.t("bonus.cgn.detail.cta.buyers"),
```

Comment:
```suggestion
                  label: canDisplayEycaDetails ? I18n.t("bonus.cgn.detail.cta.buyers") : I18n.t("bonus.cgn.detail.cta.discover"),
```
Since in [the designed page](https://www.figma.com/file/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?type=design&node-id=3380-36724&mode=design&t=iFxqKTacrUK5cuId-4) when there is no EYCA card it should only show the “Scopri le opportunità” label, what do you think putting this control on this label then declaring in the locales files this new key `discover` with the indicated value?